editor/src/components/editor/actions/actions.tsx

```diff
@@ -615,6 +615,13 @@ export function editorMoveMultiSelectedTemplates(
   editor: EditorModel
   newPaths: Array<ElementPath>
 } {
+  if (newParent == null) {
+    return {
+      editor: editor,
+      newPaths: [],
+    }
+  }
```

Comment:
The fact that this wasn't early returning before makes me very suspicious

Reply:
`getReparentOutcome` used to add a default for this "Cater for something being reparented to the canvas", I removed it because it's essentially feature logic tucked away in a library function

Reply:
Interesting, it looks like that was never the case anyway.